Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN if [ "$IS_PROD" = "true" ]; then \
         poetry install --only main --no-root; \
     else \
         poetry install --no-root; \
+        playwright install --with-deps chromium; \
```

Comment:
Switching functional tests from Firefox to Chromium due to current browser market shares